### PR TITLE
feat: hovers

### PR DESCRIPTION
**Pull request by hacdias**
Hover classes for color and background. Could you check if everything's alright and if so, publish a new version? 😄 

/cc @olizilla

### CODE REVIEW

File: build/theme.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ function buildThemeCss () {
     return `.border-${name} { border-color: ${theme.colors[name]}; }`
   })
 
+  const hover = Object.keys(theme.colors).map(name => {
+    return `.hover-${name} { transition: .2s ease color; }
```
Comment (olizilla) [inline build/theme.js]: `color` should be `${theme.colors[name]}`

File: build/theme.js [hunk 2]
```diff
@@ -20,6 +20,16 @@ function buildThemeCss () {
...
+      .hover-${name}:hover { color: ${theme.colors[name]}; }`
+  })
+
+  const hoverBg = Object.keys(theme.colors).map(name => {
+    return `.hover-bg-${name} { transition: .2s ease background-color; }
```
Comment (olizilla) [inline build/theme.js]: Same sort of thing here

File: build/theme.js (hunk 1 above)
Comment (hacdias) [reply build/theme.js]: What? Isn't it already?

image

File: build/theme.js (hunk 2 above)
Comment (hacdias) [reply build/theme.js]: I think it is done in the line 30

File: build/theme.js (hunk 1 above)
Comment (olizilla) [reply build/theme.js]: 😕 I totally misread it! you are right. 👍 

As much as possible, single purpose atoms should do 1 thing, so this kind of thing might be expressed as

```css
.transition-color {
  transition: .2s ease color;
}
.hover-red:hover { color: red;}
```

Comment (olizilla) [reply build/theme.js]: or just start with a single

```css
.transition-all {
  transition: .2s ease all;
}
```
until you find a need more fine-grained control.

**Comment by hacdias** [comment]
@olizilla take a look 😄

**Pull request merged**